google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Job.java

```diff
@@ -272,13 +275,40 @@ public TableResult getQueryResults(QueryResultsOption... options)
     }
 
     TableId table = ((QueryJobConfiguration) getConfiguration()).getDestinationTable();
-    // TODO(pongad): merge options?
+
+    List<QueryResultsOption> waitOptions = new ArrayList<>();
+    waitOptions.add(QueryResultsOption.pageSize(0L));
+    List<TableDataListOption> listOptions = new ArrayList<>();
```

Comment:
I'm somewhat concerned by this. If/When we eventually move to streaming RPC, most of these options will be useless.

Reply:
That's going to be a problem across languages. We'll figure out how to transition if/when we get a streaming RPC API for this.

Reply:
SGTM